tests/Transfer/Commons/Tests/Doctrine/DbalAdapterTest.php

```diff
@@ -41,7 +41,7 @@ public function testHandle()
         $this->assertEquals(
             array(
                 array(
-                    'id' => 1,
+                    'id'   => 1,
```

Comment:
I have now configured StyleCI not to complain about arrow alignments. This should be reverted since editing files not related to own PR usually should be avoided.